### PR TITLE
8273333: Zero should warn about unimplemented -XX:+LogTouchedMethods

### PR DESCRIPTION
**Pull request by shipilev**
Currently Zero VM does not implement -XX:+LogTouchedMethods runtime features, which makes `runtime/CommandLine/PrintTouchedMethods.java` tier1 test fail with it.

Instead of implementing it (risking performance and rare bugs), Zero should just reject the option. 

Additional testing:
 - [x] Eyeballing failure message
 - [x] Affected test does not run with Zero anymore
 - [x] Affected test still runs with Server

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8273333](https://bugs.openjdk.java.net/browse/JDK-8273333): Zero should warn about unimplemented -XX:+LogTouchedMethods

### Reviewers
 * [Yumin Qi](https://openjdk.java.net/census#minqi) (@yminqi - **Reviewer**) ⚠️ Review applies to 9e0a3218b6a3b4795aacd7b0fdf674fab08b5650
 * [David Holmes](https://openjdk.java.net/census#dholmes) (@dholmes-ora - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5363/head:pull/5363` \
`$ git checkout pull/5363`

Update a local copy of the PR: \
`$ git checkout pull/5363` \
`$ git pull https://git.openjdk.java.net/jdk pull/5363/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5363`

View PR using the GUI difftool: \
`$ git pr show -t 5363`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5363.diff">https://git.openjdk.java.net/jdk/pull/5363.diff</a>

</details>

### CODE REVIEW

**Review by yminqi — APPROVED** [review]
LGTM.

File: test/hotspot/jtreg/runtime/CommandLine/PrintTouchedMethods.java [hunk 1]
```diff
@@ -23,7 +23,8 @@
 
 /*
  * @test
- * @bug 8025692
+ * @requires vm.flavor != "zero"
+ * @bug 8025692 8273333
```
Comment (dholmes-ora) [inline test/hotspot/jtreg/runtime/CommandLine/PrintTouchedMethods.java:27]: Nit: these lines tend to be in the other order

**Review by dholmes-ora — APPROVED** [review]
Looks fine.

One style nit.

Thanks,
David

File: test/hotspot/jtreg/runtime/CommandLine/PrintTouchedMethods.java (hunk 1 above)
Comment (shipilev) [reply test/hotspot/jtreg/runtime/CommandLine/PrintTouchedMethods.java]: All right, changed.

**Review by dholmes-ora — APPROVED** [review]

**Comment by shipilev** [comment]
Thanks for reviews!

/integrate

**Pull request closed without merging**